.github/workflows/conda-envs/oldest-test-requirements.txt

```diff
@@ -1,5 +1,5 @@
 ase==3.16.0
-cmake==3.12.0
+cmake==3.13.0
```

Comment:
Why was this bumped?

Reply:
This was needed because of a change made in #950 (calling `target_link_options`). I don't remember why it didn't get caught in that PR, but I'm fixing it here.